app/services/user.js

```diff
@@ -49,3 +50,9 @@ exports.createUser = user =>
       }
       throw databaseError(error);
     });
+
+exports.updateUserRole = user =>
+  user.save().catch(error => {
```

Comment:
This method don't make sense, name tells something and implementation does something different